### ISSUE TITLE
Add 3 noteworthy (most-viewed?) video links to each youtube channel description

### ISSUE THREAD

**Issue by filfreire** [filfreire/awesome-game-essays#14]
We could add 3 noteworthy (maybe the most-viewed/popular?) video links to each youtube channel description, in order for people to directly find out the most popular videos of each channel?

How we can do this:

```
- [SomeYoutubeChannel](link) - this is a description for the channel. (Most popular videos: [1](link), [2](link), [3](link))
```

Which would look like:
- [SomeYoutubeChannel](link) - this is a description for the channel. (Most popular videos: [1](link), [2](link), [3](link))

**Comment by jmromanos** [comment]
But maybe those videos are not the most valuable ones.

**Comment by filfreire** [comment]
@jmromanos you're right. So we can follow some sort of peer-review aproach. If at least 2-3 people agree with which are the most valuable videos, those are the ones that end up in the list.
